### ISSUE TITLE
CBLS Wetlab pH probe 

### ISSUE THREAD

**Issue by zdellaert** [Putnam-Lab/Lab_Management#93]
The pH probe on Probe Set 1 of the Orion meter has been finnicky for a while, but there is clearly a bad connection in the wire that is making it currently unreliable for measurements. Since the QT corals are now in BT1/2, we have taken the pH probe from Probe Set 2 and attached it to the Set 1 Orion for the time being. We redid the Tris cal for this new configuration. We can attempt to repair the probe, continue with the current configuration, or order a new one.

**Comment by hputnam** [comment]
@zdellaert the probes are ~$400 have have a shelf life, so we can wait until needed. Is it this one? https://www.fishersci.com/shop/products/mettler-toledo-expert-series-ph-ph-atc-combination-electrodes-6/01911461

**Comment by zdellaert** [comment]
Correct, and sounds good! Just wanted a record of the swap. I will also work on troubleshooting it in the meantime.
